### PR TITLE
Release npm package from travis

### PR DESCRIPTION
**Pull request by nguyenchr**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,3 +13,14 @@ addons:
     packages:
       - g++-4.8
```
Comment (rprieto) [inline .travis.yml]: Do we still need this?

Comment (nguyenchr) [reply .travis.yml]: @rprieto I'll remove it and see if the build passes

Comment (nguyenchr) [reply .travis.yml]: @rprieto the build passes 👍

**Comment by rprieto** [comment]
Nice work, would merge again! 💯

**Pull request merged**